actstream/transformers.py

```diff
@@ -0,0 +1,5 @@
+class DefaultVerbTransformer(object):
+    verb_map = {}
+
+    def trans(self, value):
```

Comment:
I don't think `trans` is a great method name, as it's an unnecessary abbreviation. I'd like `transform()` a lot better. Please add a minimal docstring as well, explaining what `value` is and what is returned.